src/iso.c

```diff
@@ -274,9 +275,12 @@ static BOOL check_iso_props(const char* psz_dirname, int64_t file_length, const
 			}
 		}
 
-		// Check for ReactOS' setupldr.sys anywhere
-		if ((img_report.reactos_path[0] == 0) && (safe_stricmp(psz_basename, reactos_name) == 0))
-			static_strcpy(img_report.reactos_path, psz_fullpath);
+		// Check for ReactOS presence
+		if (img_report.reactos_path[0] == 0) {
+			if ((safe_stricmp(psz_basename, reactos_name) == 0) ||
+				(safe_stricmp(psz_basename, reactos_old_name) == 0))
```

Comment:
It needs also tabs for the sub-level vertical alignment of the second line of the if-condition?